code/modules/admin/bccm/bccm.dm

```diff
@@ -0,0 +1,387 @@
+// BCCM (Ban Counter Counter Measures system) ((Name subject to change)), originally inspired by EAMS (Epic Anti-Multiaccount System), by Epicus
+//version 1.0.2
+
+/datum/bccm_info
+	var/is_loaded = FALSE
+	var/is_whitelisted = FALSE
+
+	var/ip
+	var/ip_as
+	var/ip_mobile
+	var/ip_proxy
+	var/ip_hosting
+
+/client
+	var/datum/bccm_info/bccm_info = new
+
+SUBSYSTEM_DEF(bccm)
+	name = "BCCM"
+	init_order = SS_INIT_BCCM
+	flags = SS_NO_FIRE
+
+	var/max_error_count = 4
+
+	var/is_active = FALSE
+	var/error_counter = 0
+
+	var/list/tgui_panel_asn_data = list()
+	var/list/tgui_panel_wl_data = list()
+
+	var/list/client/postponed_client_queue = new
```

Comment:
I'd much rather subsystem definitions be in `code/controllers/subsystems` for consistency

Reply:
basically move this file there frankly